web3.js/rollup.config.js

```diff
@@ -69,8 +69,34 @@ function generateConfig(configType, format) {
   switch (configType) {
     case 'browser':
       switch (format) {
-        case 'esm': {
+        case 'iife': {
+          config.external = ['http', 'https'];
+
           config.output = [
+            {
+              file: 'lib/index.iife.js',
+              format: 'iife',
+              name: 'solanaWeb3',
+              sourcemap: true,
+            },
+            {
+              file: 'lib/index.iife.min.js',
+              format: 'iife',
+              name: 'solanaWeb3',
+              sourcemap: true,
+              plugins: [terser({mangle: false, compress: false})],
+            },
+          ];
+
+          break;
+        }
```

Comment:
Here we move the `iife` case, and simply default to the common case, which now includes both CJS and ESM builds.